app.py

```diff
@@ -33,13 +29,45 @@
 @app.route("/bibliography")
 @requires_db
 async def list_bibliographies(db, request):
-    to_return = [{
-        "eid": bib.eid,
-        "description": bib.description,
-        "created": bib.created,
-        "modified": bib.modified
-    } for bib in Bibliography.list(db)]
-    return json(to_return)
+    bibliographies = [
+        {
+            "eid": bib.eid,
+            "description": bib.description,
+            "created": bib.created,
+            "modified": bib.modified
+        }
+        for bib in Bibliography.list(db)
+    ]
+    return json(bibliographies)
+
+
+@app.route("/bibliography_eid")
```

Comment:
Let's use a request parameter instead of a query parameter, `@app.route('/bibliography/<eid>')...` as per this docs http://sanic.readthedocs.io/en/latest/sanic/routing.html#request-parameters.